cip-0009.md

```diff
@@ -65,20 +54,20 @@ The Memo is usually used to identify transactions to merchants and exchanges: wh
 
 # Specification
 
-The enhanced send, detailed in this proposal, utilizes the existing asset send message data structure with some additional elements: a message type range, receiving address and memo.  
+The enhanced send, detailed in this proposal, utilizes the existing asset send message data structure with some additional elements: a single byte message ID (see CIP11), receiving address and memo.  
 
-*   New Message ID Range: ````02000000```` : ````02ffffff````, data within the three byte message ID value range is referenced as the first three bytes of the receiving address
+*   New Message ID: ````0x02````
 *   Receiving Address (21 bytes, first 3 stored in the message type value range): Base58->Hex, no checksum. For converting address from Base58->Hex and vice versa, see http://lenschulwitz.com/base58
```

Comment:
Remove  `, first 3 stored in the message type value range`

Change `Base58->Hex, no checksum` to `bitcoin public key hash, no checksum`.

Change `Hex` to `binary`